docs/start/overview.md

```diff
@@ -761,7 +761,52 @@ See:
 
 ## Location State
 
-<docs-info>TODO:</docs-info>
+React Router can read and modify the [location state][locationstate], which is a part of [`location`][location] object. You can get the [`location`][location] object using `useLocation`.
+
+```jsx lines=[2,5]
+function App() {
+  const location = useLocation();
+
+  //get location state object from location object
+  const locationState = location.state;
+
+  return (
+    <p>
+      The current location state value is:
+      {locationState.someValue}
+    </p>
+  );
+}
+```
+
+You can use `<Link state>` or `useNavigate` to change the location state.
```

Comment:
In other parts of this document internal APIs, like hooks, are usually linked in the "See" section. Additionally, I don't think changing `usenavigate` to `use-navigate` is a good idea, since it will break other parts of the doc that use it.

Reply:
We try to link as much as possible on the places where things are mentioned, hence why we should add a link in this place